packages/playwright/src/worker/testInfo.ts

```diff
@@ -45,6 +45,7 @@ export interface TestStepInternal {
   error?: TestInfoError;
   infectParentStepsWithError?: boolean;
   box?: boolean;
+  isSoft?: boolean;
```

Comment:
Is this the same flag as `infectParentStepsWithError`?

Reply:
Nope, that one changes dynamically.